contracts/evoting/evoting.go

```diff
@@ -869,18 +861,23 @@ func (e evotingCommand) manageAdminList(snap store.Snapshot, step execution.Step
 }
 
 // isAdmin Check whether a user is in an Admin List
-func (e evotingCommand) isAdmin(form types.AdminList, txPerformingUser string) (bool, error) {
+func (e evotingCommand) isAdmin(snap store.Snapshot, txPerformingUser string) (bool, types.AdminList, error) {
 	// If it found the AdminList
 	// Check that the performing user is Admin
+	form, err := e.getAdminList(snap)
+	if err != nil {
+		return false, types.AdminList{}, err
+	}
+
```

Comment:
I would add the error checking here (if list cannot be retrieved because of an error) instead of doing it seperately in the calling functions, since retrieving the list successfully is part of this functions functionality now